### PR TITLE
Feature/load pie chart

### PR DESCRIPTION
**Pull request by Firefight-Rhizome**
Loading pie chart with data based on delegated preps.
slight change to how the we build the typescript class objects. I found it easier to work with.

### CODE REVIEW

File: src/app/services/icon-contract/icon-contract.service.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export class IconContractService {
     preps.totalDelegated = this.toBigInt(response.totalDelegated);
     preps.totalStake = this.toBigInt(response.totalStake);
     preps.startRanking = this.toInt(response.startRanking);
-    preps.preps = [response.preps.length];
+    preps.preps = [] = [];
```
Comment (salvation-al) [inline src/app/services/icon-contract/icon-contract.service.ts]: looks weird

Comment (salvation-al) [reply src/app/services/icon-contract/icon-contract.service.ts]: why [] = [] ???

Comment (Firefight-Rhizome) [reply src/app/services/icon-contract/icon-contract.service.ts]: might not be needed, I didnt test it without it..ive always used it that way.

Comment (Firefight-Rhizome) [reply src/app/services/icon-contract/icon-contract.service.ts]: Updated

**Pull request merged**